src/N3Parser.js

```diff
@@ -266,6 +266,18 @@ export default class N3Parser {
     case 'abbreviation':
       this._predicate = this.ABBREVIATIONS[token.value];
       break;
+    case 'literal':
+      if (!this._n3Mode)
+        return this._error('Unexpected literal', token);
+
+      if (token.prefix.length === 0) {
```

Comment:
Maybe a comment as to why we are checking prefix length here?

Reply:
Honestly I can't figure it out 😅

Reply:
Well that's a problem - I stole the logic directly from here in `_readObject` https://github.com/rdfjs/N3.js/blob/b1346130019e8b6b3094565275fba1acb10f771c/src/N3Parser.js#L314-L323.

Reply:
I'll copy over the comments there though